crates/pixi_consts/src/consts.rs

```diff
@@ -36,13 +38,39 @@ pub const _CACHED_BUILD_ENVS_DIR: &str = "cached-build-envs-v0";
 pub const CACHED_BUILD_TOOL_ENVS_DIR: &str = "cached-build-tool-envs-v0";
 pub const CACHED_GIT_DIR: &str = "git-cache-v0";
 
+pub const CONFIG_DIR: &str = match option_env!("PIXI_CONFIG_DIR") {
+    Some(dir) => dir,
+    None => "pixi",
+};
+pub const PROJECT_LOCK_FILE: &str = match option_env!("PIXI_PROJECT_LOCK_FILE") {
+    Some(file) => file,
+    None => "pixi.lock",
+};
+pub const PIXI_DIR: &str = match option_env!("PIXI_DIR") {
+    Some(dir) => dir,
+    None => ".pixi",
+};
+
+lazy_static! {
+    /// The default channels to use for a new project.
+    pub static ref DEFAULT_CHANNELS: Vec<String> = match option_env!("PIXI_DEFAULT_CHANNELS") {
+        Some(channels) => channels.split(',').map(|s| s.to_string()).collect(),
+        None => vec!["conda-forge".to_string()],
+    };
+
+    /// The name of the binary.
+    pub static ref PIXI_BIN_NAME: String = std::env::args().next()
```

Comment:
perhaps we should move this out of the consts crate since its not really a const.

Reply:
that's true. maybe into pixi_utils?

Reply:
Personally, I am fine with having that in `consts.rs` since it is determined during compilation time.
If @baszalmstra prefers it, `pixi_utils` sounds good to me as well.

Reply:
It uses std::env::args().next() and resolves at runtime. Its not really a constant. if/when we turn pixi_consts into a published crate it will yield different values.

Pixi_utils sounds good to me! Or just keep it in pixi itself if its only used there?

Reply:
It's currently used in pixi's top level mod.rs (for error messages) and completion.rs (for completions) so no clear one place the most sense in pixi itself so I'll move to pixi_utils.

Reply:
moved to pixi_utils